airbyte-webapp/src/scss/_colors.scss

```diff
@@ -0,0 +1,55 @@
+$primaryColor: #615eff;
```

Comment:
Could we still add a comment here, that this is currently a duplication of https://github.com/airbytehq/airbyte/blob/master/airbyte-webapp/src/theme.ts and they should always edited at the same time.